linkerd/app/outbound/src/opaq/logical/route.rs

```diff
@@ -78,13 +97,21 @@ where
                 // consideration, so we must eagerly fail requests to prevent
                 // leaking tasks onto the runtime.
                 .push_on_service(svc::LoadShed::layer())
+                .push_filter(|rt: Self| -> Result<_, Error> {
+                    if rt.params.forbidden {
+                        Err(Errno::from(Code::ECONNREFUSED).into())
```

Comment:
Not sure this will actually work for Windows. We seem to not have this enum there...